packages/syntax-floating-components/src/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,160 @@
+        "bottom-start",
+        "bottom-end",
+        "left",
+        "left-start",
+        "left-end",
+      ],
+      table: {
+        defaultValue: { summary: "right" },
+      },
+    },
+    strategy: {
+      control: { type: "select" },
+      options: ["absolute", "fixed"],
+      defaultValue: "absolute",
+      table: {
+        defaultValue: { summary: "absolute" },
+      },
+    },
+    children: {
+      control: { type: "text" },
+      description: "The string value to show on the tooltip content",
+      defaultValue: "This is a button",
+      table: {
+        type: { summary: "string" },
+        defaultValue: { summary: "This is a button" },
+      },
+    },
+  },
+  tags: ["autodocs"],
+} as Meta<typeof Tooltip>;
+
+export const Default: StoryObj<typeof Tooltip> = {
+  args: {
+    delay: 0,
+    placement: "right",
+    initialOpen: true,
+    strategy: "absolute",
+    children: "This is a book",
+  },
+  render: ({ delay, placement, initialOpen, strategy, children }) => (
+    <Tooltip
+      delay={delay}
+      placement={placement}
+      initialOpen={initialOpen}
+      strategy={strategy}
+    >
+      <TooltipTrigger>
+        <img
+          src={image as string}
+          alt="info icon"
+          style={{ width: "20px", height: "20px" }}
+        />
+      </TooltipTrigger>
+      <TooltipContent>
+        <Typography size={100} color="white">
+          {children}
+        </Typography>
+      </TooltipContent>
+    </Tooltip>
+  ),
+};
+
+export const ControlledTooltip = () => {
+  const [open, setOpen] = useState(false);
+  const scrollableRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (scrollableRef.current) {
+      scrollableRef.current.scrollTop =
+        scrollableRef.current.scrollHeight / 2 -
+        scrollableRef.current.offsetHeight / 2;
+    }
+  }, [scrollableRef]);
+
+  return (
+    <div
+      ref={scrollableRef}
+      style={{
+        overflowY: "auto",
+        width: "400px",
+        height: "300px",
+      }}
+    >
+      <div
+        style={{
+          marginTop: "300px",
+          marginBottom: "300px",
+          display: "flex",
+          justifyContent: "center",
+        }}
+      >
+        <Tooltip open={open} onOpenChange={setOpen}>
+          <TooltipTrigger>
+            <button onClick={() => setOpen((v) => !v)}>My Trigger</button>
+          </TooltipTrigger>
+          <TooltipContent>
+            <Typography size={100} color="white">
+              This is a book and a really long sentence.
+            </Typography>
+            <a
+              href="http://localhost:6006/?path=/docs/floating-components-tooltip--docs"
+              style={{
+                textUnderlinePosition: "under",
+                color: "white",
+                paddingTop: "8px",
+                fontSize: "12px",
+                fontWeight: 510,
+                lineHeight: "14px",
+              }}
+            >
+              Learn more
```

Comment:
@JFChoe could we also wrap this in a `<Typography />` component? 
![image](https://github.com/Cambly/syntax/assets/127199/3a27b89b-937c-4570-b4df-6e6b99864462)

Then you should not have to set the fontweight / lineheight and other props